### ISSUE TITLE
Comment in chained method call pulled to previous line, indentation lost

### ISSUE THREAD

**Issue by mafredri** [prettier/prettier#1958]
While reporting #1957 I noticed that given two chained method calls and a comment in-between, the comment was pulled to the previous line and indentation for the continuation was lost.

**Input:**

```javascript
angular.module('AngularAppModule')
  // Hello, I am comment.
  .constant('API_URL', 'http://localhost:8080/api');
```

**Output:**

```javascript
angular.module('AngularAppModule')// Hello, I am comment.
.constant('API_URL', 'http://localhost:8080/api');
```

**Expected:**

Either no difference between input and output, or:

```javascript
angular
  .module('AngularAppModule')
  // Hello, I am comment.
  .constant('API_URL', 'http://localhost:8080/api');
```

Reproduction can be found [here](https://prettier.github.io/prettier/#%7B%22content%22%3A%22angular.module('AngularAppModule')%5Cn%20%20%2F%2F%20Hello%2C%20I%20am%20comment.%5Cn%20%20.constant('API_URL'%2C%20'http%3A%2F%2Flocalhost%3A8080%2Fapi')%3B%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Atrue%2C%22trailingComma%22%3A%22es5%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22babylon%22%2C%22semi%22%3Atrue%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%7D%7D).

Problem goes away after a third chained method call is added.

**Sidenote:** Sorry for the issue bombardment, I really appreciate all the work you put into prettier and I think it' awesome! Keep up the good work :-).

**Comment by vjeux** [comment]
Thanks for the bug report, we should fix this.

Keep your issues coming, they are great!

**Comment by vjeux** [comment]
I just published 1.4.3 which fixes it. Thanks for reporting it!
